src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,18 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import './PostInfo.scss';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ post }) => {
+  const { title, body, comments, user } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+        <UserInfo user={user} />
+      </div>
+      <p className="PostInfo__body">{body}</p>
+      {CommentList && <CommentList comments={comments} />}
```

Comment:
```suggestion
      <CommentList comments={comments} />
```
looks like redundant check